### DISCUSSION TITLE
Project roadmap week 2

### DISCUSSION THREAD

**Discussion by TarunavBA** [TarunavBA/Social-Network#26] [General]
First hey @Spratiher9 welcome to this project and thanks for contributing to this project. You ca convert this project into a django project and start working on it I will also join you.

**Comment by TarunavBA** [comment]
Hey @Spratiher9 can you add a `pipfile` and a `pipfile.lock` by installing pipenv by command `pip install pipenv`?

**Reply by Spratiher9** [reply]
Okay let me look at this

**Comment by TarunavBA** [comment]
@Spratiher9 refer this https://blog.pyplane.com/blog/social-network-project-in-django/ series for more information

**Comment by Spratiher9** [comment]
@TarunavBA why specifically Django and not FastAPI.

**Reply by TarunavBA** [reply]
@Spratiher9 it's because django is built for robust applications like social media. So this is the main reason why I chose django.

**Comment by TarunavBA** [comment]
@Spratiher9 Use [this](https://gist.github.com/rxaviers/7360908) list of emojis while commiting.
